controllers/storagecluster/cephcluster.go

```diff
@@ -492,6 +496,21 @@ func newCephCluster(sc *ocsv1.StorageCluster, cephImage string, kmsConfigMap *co
 		MaxLogSize:  &maxLogSize,
 	}
 
+	strVal := util.MustGetEnv(desiredCephxKeyGenEnvVarName)
+	intVal, err := strconv.Atoi(strVal)
+	if err != nil {
+		panic(fmt.Sprintf("could not convert the value %q of env var %q", strVal, desiredCephxKeyGenEnvVarName))
```

Comment:
As stated in the prev PR please return an error as this does not reflect a coding error (a build time assertion) but a run time assertion